src/snyk/extension.ts

```diff
@@ -231,7 +231,7 @@ class SnykExtension extends SnykLib implements IExtension {
             analytics.logIssueIsViewed({
               ide: 'Visual Studio Code',
               issueId: suggestion.id,
-              issueType: 'Code Security Vulnerability',
+              issueType: suggestion.isSecurityType ? 'Code Security Vulnerability' : 'Code Quality Issue',
```

Comment:
@michelkaporin, is any way to test this change?

Reply:
@pavel-snyk I'd love having integration tests for analytics generally but it's out of scope of this PR. I've verified the change in Iteratively Live Stream.